src/nes.nim

```diff
@@ -36,4 +36,4 @@ proc run*(nes: NES, seconds: float) =
     cycles -= nes.step()
 
 proc buffer*(nes: NES): var Picture =
-  nes.ppu.front
+  nes.ppu.front
```

Comment:
please add a newline here and in the other files, rest of the change looks fine.

Reply:
@def- done